1266/en.md

```diff
@@ -0,0 +1,119 @@
+# LightOj 1266 - Points in Rectangle
+[Problem Statement](http://lightoj.com/volume_showproblem.php?problem=1266)  
+### Summary
+You will be given two type of query and the range of query is 1<=q<=3000. Two type of query is -
+1. 0 x y  that means add a point in (x,y) co-ordinate if it doesn't already exist.
+2. 1 x1 y1 x2 y2 - you have to find the number of point exist in this range where     (x1,y1) is lower left corner of rectangle and (x2,y2) is upper right corner.
```

Comment:
I think we can ditch the _Problem Statement_ URL and the _Summary_ section, since the problem statement itself describes it better.

Reply:
thank you for your kind suggestion. i will change it as fast as possible